fluent-dom/src/localization.js

```diff
@@ -1,28 +1,44 @@
 /* eslint no-console: ["error", { allow: ["warn", "error"] }] */
 /* global console */
 
+import CachedIterable from '../../fluent/src/cached_iterable';
```

Comment:
Because there's no glue code in Localization.jsm. As Mossop noted, we never allow for non-cached iterable in it.